app/src/main/kotlin/com/simplemobiletools/calendar/pro/activities/MainActivity.kt

```diff
@@ -589,6 +593,37 @@ class MainActivity : SimpleActivity(), RefreshRecyclerViewListener {
         }
     }
 
+    private fun addBirthdaysAnniversaries() {
+        if (!hasPermission(PERMISSION_READ_CONTACTS)) {
+            return
+        }
+
+        val reminders = arrayListOf(REMINDER_OFF, REMINDER_OFF, REMINDER_OFF)
```

Comment:
the user can already select reminder at importing birthdays and anniversaries, lets reuse the values he selected here too

Reply:
People will expect having the last used reminders there. Just store the 3 reminders in some new shared pref config field.